app/cafe.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError,
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(
+                "Visitor is not vaccinated."
+            )
+
+        expiration_date = visitor["vaccine"].get("expiration_date")
+
+        if expiration_date < datetime.date.today():
+            raise OutdatedVaccineError(
+                "Visitor vaccine is outdated."

```

Comment:
The error message here should match the default message in the `OutdatedVaccineError` class, which is "Visitor's vaccine is outdated.". Consider updating the message to maintain consistency with the error class definition.